.ci/packaging.groovy

```diff
@@ -109,7 +109,8 @@ pipeline {
           }
           post {
             failure {
-              notifyStatus(subject: "[${env.REPO}@${env.BRANCH_NAME}] package failed")
+              notifyStatus(subject: "[${env.REPO}@${env.BRANCH_NAME}] package failed.",
+                           body: 'Contact the Productivity team if you need further assistance')
```

Comment:
Should we add channel names?

Reply:
notifyStatus reports by email and slack, so do you prefer the slack channels?

Reply:
ah I missed that. Maybe we could include both? but don't let my review block the merging of this PR.

Reply:
Let me work on this

Reply:
I just added the slack channel -> https://github.com/elastic/apm-server/pull/7723/commits/c8fa511919e3c3dc5c5874bb2b84cfc8b477af60

Let's try with the slack rather than the email to avoid exposing further details, what do you think?

Reply:
Sounds good, thank you!